diesel_cli/tests/database_reset.rs

```diff
@@ -170,3 +170,23 @@ fn reset_sanitize_database_name() {
         result.stdout()
     );
 }
+
+#[test]
+fn reset_updates_schema_if_config_present() {
+    let p = project("reset_updates_schema_if_config_present")
+        .folder("migrations")
+        .file(
+            "diesel.toml",
+            r#"
+[print_schema]
```

Comment:
I don't think TOML cares about whitespace does it? Can we have this indented normally?

Reply:
You are correct, I changed the indentation in `migration_run_updates_schema_if_config_present` as well.